app/validators/belongs_to_host_taxonomy_validator.rb

```diff
@@ -9,17 +9,14 @@ def validate_each(record, attribute, value)
     return unless record.host.present? && value.present?
 
     host_taxonomy = record.host.public_send(taxonomy)
-    attribute_taxonomies = value.public_send(taxonomy.to_s.pluralize.to_sym)
+    return if host_taxonomy.nil?
```

Comment:
I believe we never care about cases where taxonomy is not defined on the host, right? it should be validated on the host so the host shouldn't be saved -> the dependendant model shouldn't eighter, right?

Reply:
Yeah, we now have taxonomies always enabled.